mdn/yari
# chore(glean): add page's UTM parameters to pings

## Summary



https://mozilla-hub.atlassian.net/browse/MP-545
https://bugzilla.mozilla.org/show_bug.cgi?id=1851150

### Problem



We can't easily attribute traffic which adds utm parameters through Glean.

### Solution



Add utm parameters to our Glean pings.

---

## How did you test this change?

- `yarn && yarn dev`
- visit a page locally, appending utm parameters, e.g. the ones given in the example [on wikipedia](https://en.wikipedia.org/wiki/UTM_parameters)
- observe ping: https://debug-ping-preview.firebaseapp.com/pings/mdn-dev/4ada8297-949e-4904-9bc8-39d0d8922b0f

@fiji-flo after sending this ping, I realised we already sorta send utm parameters to glean, through the `page.path` metric - since we'll now be sending these in a structured form, should I wipe `utm_` parameters out of the url we send in that metric (or even not send any query parameters in `page.path` at all, to keep it "clean"?)
